generic_io/rs485/firmware.lua

```diff
@@ -0,0 +1,89 @@
+local config = require('enapter.ucm.config')
+local qmodbus = require('enapter.ucm.qmodbus')
+
+local rs485_configured = false
+local rs485_not_configured_err = 'rs485 is not properly configured'
+
+BAUD_RATE = 'baud_rate'
+DATA_BITS = 'data_bits'
+PARITY = 'parity'
+STOP_BITS = 'stop_bits'
+BUFFER_SIZE = 'buffer_size'
+
+function main()
+  config.init({
+    [BAUD_RATE] = { type = 'number', required = true },
+    [DATA_BITS] = { type = 'number', required = true },
+    [PARITY] = { type = 'string', required = true },
+    [STOP_BITS] = { type = 'number', required = true },
+    [BUFFER_SIZE] = { type = 'number', default = 4096 },
+  }, {
+    after_write = setup_rs485,
+  })
+
+  enapter.register_command_handler('read', cmd_read)
+  enapter.register_command_handler('write', cmd_write)
+
+  scheduler.add(1000, send_telemetry)
+  scheduler.add(30000, send_properties)
+end
```

Comment:
Is `setup_rs485` missing here? So that it gets set up after reboot.

Reply:
Yes, I found it during testing. Updated version is pushed.